aws/ami/files/scylla_install_ami

```diff
@@ -60,64 +68,106 @@ if __name__ == '__main__':
 
-    if args.repo_for_install:
-        run('curl -L -o /etc/yum.repos.d/scylla_install.repo {REPO_FOR_INSTALL}'.format(REPO_FOR_INSTALL=args.repo_for_install))
+        if args.localrpm:
+            rpms = glob.glob('{}/scylla*.*.rpm'.format(homedir))
+            run('yum install -y {}'.format(' '.join(rpms)))
+        else:
+            run('yum install -y {0}-machine-image {0}-debuginfo'.format(args.product))
 
-    if args.localrpm:
-        rpms = glob.glob('{}/scylla*.*.rpm'.format(homedir))
-        run('yum install -y {}'.format(' '.join(rpms)))
+        if args.repo_for_install:
+            os.remove('/etc/yum.repos.d/scylla_install.repo')
+        if args.repo_for_update:
+            run('curl -L -o /etc/yum.repos.d/scylla.repo {REPO_FOR_UPDATE}'.format(REPO_FOR_UPDATE=args.repo_for_update))
     else:
-        run('yum install -y {0}-machine-image {0}-debuginfo'.format(args.product))
+        run('apt-get update')
+        run('apt-get upgrade -y')
+        run('apt-get install gdebi-core -y')
+
+        if args.repo_for_install:
+            run('curl -L -o /etc/apt/sources.list.d/scylla_install.list {REPO_FOR_INSTALL}'.format(REPO_FOR_INSTALL=args.repo_for_install))
+            run('apt-get update --allow-insecure-repositories -y')
 
-    if args.repo_for_install:
-        os.remove('/etc/yum.repos.d/scylla_install.repo')
-    if args.repo_for_update:
-        run('curl -L -o /etc/yum.repos.d/scylla.repo {REPO_FOR_UPDATE}'.format(REPO_FOR_UPDATE=args.repo_for_update))
+        # gdebi can install only one package at once, and dependencies should
+        # resolved on every install, so we need to manually take care
+        # dependencies between scylla packages.
+        if args.localrpm:
+            pkgs = ['-conf', '-kernel-conf', '-python3', '-server', '-server-dbg', '-node-exporter', '-jmx', '-tools-core', '-tools', '-machine-image', '']
+            for pkg in pkgs:
+                deb = glob.glob('{}/{}{}_*.deb'.format(homedir, args.product, pkg))
+                run('gdebi -n {}'.format(deb[0]))
+        else:
+            run('apt-get install -y --allow-unauthenticated {0}-machine-image {0}-debuginfo'.format(args.product))
+
+        if args.repo_for_install:
+            os.remove('/etc/apt/sources.list.d/scylla_install.list')
+        if args.repo_for_update:
+            run('curl -L -o /etc/apt/sources.list.d/scylla.list {REPO_FOR_UPDATE}'.format(REPO_FOR_UPDATE=args.repo_for_update))
 
     run('systemctl daemon-reload')
     run('systemctl enable scylla-image-setup.service')
-    with open('/etc/cloud/cloud.cfg') as f:
-        lines = f.readlines()
-    with open('/etc/cloud/cloud.cfg', 'w') as f:
-        for l in lines:
-            if not re.match(r'^ - mounts\n$', l):
-                f.write(l)
     run('/opt/scylladb/scripts/scylla_setup --ntp-domain amazon --no-coredump-setup --no-sysconfig-setup --no-raid-setup --no-io-setup --no-bootparam-setup --no-ec2-check --no-swap-setup')
```

Comment:
got 
```
==> amazon-ebs: usage: scylla_ntp_setup [-h]
==> amazon-ebs: scylla_ntp_setup: error: unrecognized arguments: --subdomain amazon
    amazon-ebs: NTP setup failed.
```
need to run without `--ntp-domain amazon`
```suggestion
    if distro == 'centos':
        run('/opt/scylladb/scripts/scylla_setup --ntp-domain amazon --no-coredump-setup --no-sysconfig-setup --no-raid-setup --no-io-setup --no-bootparam-setup --no-ec2-check --no-swap-setup')
    else:
        run('/opt/scylladb/scripts/scylla_setup --no-coredump-setup --no-sysconfig-setup --no-raid-setup --no-io-setup --no-bootparam-setup --no-ec2-check --no-swap-setup')

```

Reply:
This is because you are testing with Scylla-4.4, and I implemented on master branch.
Since https://github.com/scylladb/scylla/commit/5d527bd17e372e938a8dcb81a0d96a8374720620, this error won't occur on scylla_ntp_setup.